azimuth/plots/dataset_warnings.py

```diff
@@ -432,113 +434,69 @@ def class_representation(
 
 
 def create_histogram_mean_std(
-    train_count: np.ndarray,
-    eval_count: np.ndarray,
-    threshold_mean: float,
-    threshold_std: float,
-    train_stats: Tuple[np.ndarray, np.ndarray],
-    eval_stats: Tuple[np.ndarray, np.ndarray],
-    divergence_mean: Optional[float],
-    divergence_std: Optional[float],
-    label_name: Optional[str] = None,
+    hist_per_split: Dict[DatasetSplitName, np.ndarray],
+    agg_per_split: Dict[DatasetSplitName, Dict[Agg, float]],
+    divergence_per_agg: Optional[Dict[Agg, float]] = None,
 ) -> PlotSpecification:
     """Create the histogram traces and annotations for each label (and "all").
 
     Args:
-        train_count: Histogram values for the selected label in the train set.
-        eval_count: Histogram values for the selected label in the evaluation set.
-        threshold_mean: Alert value.
-        threshold_std: Alert value.
-        train_stats: Mean and std dev of the train.
-        eval_stats: Mean and std dev of the evaluation set.
-        divergence_mean: Difference in the mean. None means one distribution is empty.
-        divergence_std: Difference in the std dev. None means one distribution is empty.
-        label_name: Name of the label, when sending 'None', the traces are generated for 'all'.
+        hist_per_split: Histogram values for the selected label per split.
+        agg_per_split: Mean and std dev for the selected label per split.
```

Comment:
So isn't that an `agg_per_label_per_split`?
```suggestion
        agg_per_label_per_split: Mean and std dev for the selected label per split.
```